acai-ios/UIColor+.swift

```diff
@@ -0,0 +1,22 @@
+//
+//  UIColor+.swift
+//  acai-ios
+//
+//  Created by Artesia Ko on 3/7/19.
+//  Copyright © 2019 Cornell AppDev. All rights reserved.
+//
+
+import Foundation
+import UIKit
+
+extension UIColor {
+    
+    static var acaiLightGray: UIColor = UIColor(red: 199/255, green: 199/255, blue: 199/255, alpha: 1.0)
```

Comment:
I would change `static var` to `static let` for all of these since they're not subject to change.